api-reference/20 Data Visualization Widgets/dxPolarChart/1 Configuration/argumentAxis/categories.md

```diff
@@ -4,7 +4,202 @@ type: Array<Number, String, Date>
 ---
 ---
 ##### shortDescription
-Specifies the order in which arguments (categories) are arranged on the discrete argument axis.
+Specifies the order of categories on an axis of the *"discrete"* [type](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1%20Configuration/argumentAxis/type.md '/Documentation/ApiReference/Data_Visualization_Widgets/dxPolarChart/Configuration/argumentAxis/#type').
 
 ---
-If you specify the chart's data using a common array of objects, these objects may be displayed in a random order. If you set the chart's data for each series individually, the order in which the series are positioned in the [series](/api-reference/20%20Data%20Visualization%20Widgets/dxPolarChart/1%20Configuration/series '/Documentation/ApiReference/Data_Visualization_Widgets/dxPolarChart/Configuration/series/') array can also be random. In these instances, the resulting argument order on the discrete argument axis may not be appropriate. To specify the order of categories (arguments on a discrete argument axis), assign an array of category names to the **categories** property.
\ No newline at end of file
+Arguments of the `string` type on discrete axes keep the order of objects in the data source. Assign an array of properly sorted arguments to the **categories** option to sort the arguments, for example, alphabetically: 
```

Comment:
I think it is possible to inherit the description from the Chart. Of course, you should use {WidgetName} in the Chart's code in this case.